entrypoint.sh

```diff
@@ -12,20 +12,21 @@ if [[ -z "$PATHFINDER_URL" ]]; then
   exit 1
 fi
 
-if [[ -z "$SSO_REALM" ]]; then
+if [[ -z "$KEYCLOAK_REALM" ]]; then
   echo "You must provide SSO_REALM environment variable" 1>&2
   exit 1
 fi
 
-if [[ -z "$SSO_SERVER_URL" ]]; then
-  echo "You must provide SSO_SERVER_URL environment variable" 1>&2
-  exit 1
-fi
-
-if [[ -z "$SSO_CLIENT_ID" ]]; then
+if [[ -z "$KEYCLOAK_CLIENT_ID" ]]; then
```

Comment:
KEYCLOAK_CLIENT_ID and KEYCLOAK_REALM should be contained within the AUTH_REQUIRED conditional below, operator will only set these variables in the pod when AUTH_REQUIRED is true, otherwise they will be ignored but the entrypoint should not fail (AUTH_REQUIRED=false case).